skins/mettalex.scss

```
@@ -0,0 +1,10 @@
+.mtlx {
```

The css selector should similar to the file name.
```suggestion
.metallex {
```